### DISCUSSION TITLE
Extruder direction of rotation

### DISCUSSION THREAD

**Discussion by Bersaker** [Jyers/Marlin#925] [Ideas]
Hello, Is there a way to set the direction of rotation for the extruder in the menu?  Or can you only do that directly via the firmware or the cable on the connector?  It would be a good idea to be able to change this in the menu.

Tkaus

**Comment by smooreace** [comment]
Did you try setting e-steps to a negative number?

**Reply by Bersaker** [reply]
Can only be negative. up to 0 is not possible
